src/utils/FileManagementUtils.js

```diff
@@ -80,6 +81,29 @@ async function _createEmptyDatasetInCloudStorage(organizationId, parameterMetada
   return createdDataset;
 }
 
+function _forgeNameOfUploadedFile(file, parameterMetadata, defaultFileTypeFilter) {
+  const shouldRenameFile = ConfigUtils.getParameterAttribute(parameterMetadata, 'shouldRenameFileOnUpload');
+  if (shouldRenameFile == null || shouldRenameFile === false) return file?.name;
```

Comment:
You can shorten `(shouldRenameFile == null || shouldRenameFile === false)` by `!shouldRenameFile`